python/tvm/relax/frontend/onnx/onnx_frontend.py

```diff
@@ -166,6 +167,9 @@ class Div(OnnxOpConverter):
 
     @classmethod
     def _impl_v14(cls, bb, inputs, attr):
+        if all([isinstance(inp, relax.Constant) for inp in inputs]):
+            output = inputs[0].data.numpy() / inputs[1].data.numpy()
+            return relax.const(output, inputs[0].struct_info.dtype)
```

Comment:
Nit, do you think we could put this in a separate pass?

Reply:
I think in theory this could be done via foldconstants, however, I've found that taking the approach of handling constants in the importer makes dealing with models way easier. After this change, we can import pretty much all the high priority models. If constant folding and dynamic expression someday are really mature we can drop this handling.

Reply:
Let's perhaps mark at TODO there. I think it should be "properly" handled by constant folding so let's not forget about that